packages/components/avatar/src/avatar.ts

```diff
@@ -86,9 +88,14 @@ export class Avatar extends LitElement {
   /** @private */
```

Comment:
If you're rendering `<sl-tooltip>` inside this component, then this needs to use the `ScopedElementsMixin`, right?

Reply:
And also `@sl-design-system/tooltip` dependency in the `package.json` needs to be added?